automatic/apache-httpd/update.ps1

```diff
@@ -0,0 +1,37 @@
+Import-Module au
+Import-module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
+
+function global:au_AfterUpdate { Set-DescriptionFromReadme -SkipFirst 1 }
+
+function global:au_GetLatest {
+    $downloadPageUrl = 'http://www.apachehaus.com/cgi-bin/download.plx'
+    $versionRegEx = 'httpd\-([0-9\.]+)\-x\d+\-(.*)\.zip'
+    $url32 = 'https://www.apachehaus.com/downloads/httpd-$version-x86-$vcNumber.zip'
```

Comment:
I can, but as it is right now, the strings are defined in one place, on top of the function, in my opinion having them in the same place makes the code more readable.